daemon/container_operations_unix.go

```diff
@@ -1172,3 +1173,7 @@ func isLinkable(child *container.Container) bool {
 	_, ok := child.NetworkSettings.Networks["bridge"]
 	return ok
 }
+
+func errorRemovalContainer(containerID string) error {
```

Comment:
stylistic nit, all other errors start with `err<Something>`, this one `error<Something>`